customTests/custom-test-setup.sh

```diff
@@ -67,41 +67,61 @@ function install_nvbandwidth(){
 	popd
 }
 
+function check_if_command() {
+    if command -v $1 &> /dev/null; then
+        echo "$1"
+    fi
+}
+
+# Function to find clang in specified path
+function find_clang_in_path() {
+    local path="$1"
+	if [ ! -d "$path" ]; then
+		return
+	fi
+    local clang_path=$(find "$path" -name "clang" | grep '/bin/')
```

Comment:
Technically yes. I did test on our Spack images, and it return 1 line. I addressed the possibility of more lines by checking if the result was a command. It will error out if not found. This only happens during install step. I don't think this is high risk. We can always address any issues by adding clang to path.